plugin.xml

```diff
@@ -7,7 +7,7 @@
     <author>Ryan Graham</author>
     <version>${project.version}</version>
     <date>2025-06-25</date>
```

Comment:
Let's also update the 'date' field while you're changing things.

Reply:
We certainly must update it when doing a release, but I find it to be useful to update the date, to easily differentiate between various SNAPSHOT releases. It's certainly not a _must_, but it does come in handy sometimes.